cmd/kogito/resource_checks.go

```diff
@@ -10,9 +10,18 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const (
-	kogitoOperatorInstallationInstruc = "https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#installation"
-)
+// ensureProject will check if the given project is a valid string in the context and if exists in the cluster
+func ensureProject(project string) (string, error) {
+	var err error
```

Comment:
why are you setting this var here?

Reply:
copy and paste from old code. fixed, thanks!!